src/Components/Analyzers/test/ComponentParametersShouldNotBePublicTest.cs

```diff
@@ -12,7 +12,7 @@ namespace Microsoft.AspNetCore.Components.Analyzers.Test
 {
     public class ComponentParametersShouldNotBePublic : CodeFixVerifier
     {
-        static string BlazorParameterSource = $@"
+        static string ParameterSource = $@"
```

Comment:
I did a general pass at removing the word **Blazor** where it occurs and isn't specific to Blazor. There weren't that many casual references to blazor in the code.... EXCEPT in the .js side of the world where many user-facing APIs still use the term blazor for things that apply to both components and blazor. Logged https://github.com/aspnet/AspNetCore/issues/7935